apps/src/appMain.js

```diff
@@ -1,9 +1,7 @@
 var utils = require('./utils');
 var _ = utils.getLodash();
 var requiredBlockUtils = require('./required_block_utils');
-var StudioAppClass = require('./StudioApp');
-
-var studioAppSingleton = require('./base');
+var studioAppSingleton = require('./StudioApp').singleton;
```

Comment:
We could also now just start calling this studioApp instead of studioAppSingleton everywhere. Thoughts?